src/styles/annotator/notebook.scss

```diff
@@ -1,16 +1,16 @@
 @use '../variables' as var;
 @use '../mixins/molecules';
+@use "../mixins/buttons";
 
 .notebook-outer {
   box-sizing: border-box;
   position: fixed;
+  z-index: 2147483647;
```

Comment:
This does fix the PDF viewer toolbar interference (though I do get twitchy when I see CSS trying to out-z-index things). Can you add a comment here as to why this `z-index` rule was added?

Reply:
Yes, I agree it is not a very clean solution. The adder has a mechanism to compute the zIndex of the 'neighbouring' elements so it is not a hard coded value. Would you like me to investigate using the same mechanism here?

I am using the same zIndex that the sidebar uses. Do you know if there has been issues with sidebar not been visible or partially hidden because the zIndex was hard coded (and no sufficiently high)?

I added the comment you suggested.